### PR TITLE
ATS-862: Release ATS 1.3.2-A2

### PR DESCRIPTION
**Pull request by TheOutsSeeker**

### CODE REVIEW

**Review by KikoUK — APPROVED** [review]

File: helm/alfresco-content-services/6.0.N_values.yaml [hunk 1]
```diff
@@ -73,7 +73,7 @@ transformrouter:
   replicaCount: 2
   image:
     repository: quay.io/alfresco/alfresco-transform-router
-    tag: "1.0.2.1"
+    tag: "1.3.2-A2"
```
Comment (gavincornwell) [inline helm/alfresco-content-services/6.0.N_values.yaml:76]: You've updated 6.0.N for helm but not for docker compose, did you intend to do this one?

In fact, do we want to update 6.0.N at all, we normally only update latest and 6.2.N?

File: helm/alfresco-content-services/6.1.N_values.yaml [hunk 2]
```diff
@@ -73,7 +73,7 @@ transformrouter:
   replicaCount: 2
```
Comment (gavincornwell) [inline helm/alfresco-content-services/6.1.N_values.yaml:73]: Do we want to update 6.1.N at all, we normally only update latest and 6.2.N?

File: docker-compose/6.1.N-docker-compose.yml [hunk 3]
```diff
@@ -47,7 +47,7 @@ services:
 
     transform-router:
         mem_limit: 512m
-        image: quay.io/alfresco/alfresco-transform-router:1.0.2
+        image: quay.io/alfresco/alfresco-transform-router:1.3.2-A2
```
Comment (gavincornwell) [inline docker-compose/6.1.N-docker-compose.yml:50]: Do we want to update 6.1.N at all, we normally only update latest and 6.2.N?

**Review by gavincornwell — CHANGES_REQUESTED** [review]

**Review by alandavis — CHANGES_REQUESTED** [review]
As I'm going to do a A25 build of acs-packaging, I will take a look at these changes.

File: docker-compose/6.2.N-docker-compose.yml [hunk 4]
```diff
@@ -62,7 +62,7 @@ services:
           - activemq
 
     transform-core-aio:
-        image: alfresco/alfresco-transform-core-aio:2.3.6
+        image: alfresco/alfresco-transform-core-aio:2.3.8
         mem_limit: 1536m
         environment:
             JAVA_OPTS: " -XX:MinRAMPercentage=50 -XX:MaxRAMPercentage=80"
```
[the remark this answers is not in the document]
Comment (alandavis) [reply docker-compose/6.2.N-docker-compose.yml:68]: fixed

File: docker-compose/community-docker-compose.yml [hunk 5]
```diff
@@ -46,15 +46,15 @@ services:
...
         mem_limit: 1536m
         environment:
             JAVA_OPTS: " -XX:MinRAMPercentage=50 -XX:MaxRAMPercentage=80"
         ports:
             - 8090:8090
 
     share:
-        image: alfresco/alfresco-share:7.0.0-M3
+        image: alfresco/alfresco-share:7.0.0-A17
         mem_limit: 1g
```
Comment (alandavis) [inline docker-compose/community-docker-compose.yml:58]: same as acs-packaging

File: helm/alfresco-content-services/6.0.N_values.yaml (hunk 1 above)
Comment (alandavis) [reply helm/alfresco-content-services/6.0.N_values.yaml]: reverted

File: helm/alfresco-content-services/6.1.N_values.yaml (hunk 2 above)
Comment (alandavis) [reply helm/alfresco-content-services/6.1.N_values.yaml]: reverted

File: docker-compose/6.1.N-docker-compose.yml (hunk 3 above)
Comment (alandavis) [reply docker-compose/6.1.N-docker-compose.yml]: reverted

[the remark this answers is not in the document]
Comment (alandavis) [reply docker-compose/6.1.N-docker-compose.yml]: reverted

File: helm/alfresco-content-services/6.0.N_values.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (alandavis) [reply helm/alfresco-content-services/6.0.N_values.yaml]: reverted

File: helm/alfresco-content-services/6.1.N_values.yaml [hunk 6]
```diff
@@ -73,7 +73,7 @@ transformrouter:
...
   image:
     repository: quay.io/alfresco/alfresco-transform-router
-    tag: "1.0.2"
+    tag: "1.3.2-A2"
```
[the remark this answers is not in the document]
Comment (alandavis) [reply helm/alfresco-content-services/6.1.N_values.yaml]: reverted

File: helm/alfresco-content-services/6.1.N_values.yaml [hunk 7]
```diff
@@ -263,7 +263,7 @@ filestore:
   replicaCount: 1
   image:
     repository: alfresco/alfresco-shared-file-store
-    tag: "0.5.3"
+    tag: "0.12.0"
```
[the remark this answers is not in the document]
Comment (alandavis) [reply helm/alfresco-content-services/6.1.N_values.yaml]: reverted

**Comment by alandavis** [comment]
Reason the build is red is that the community content repo image that is referenced was not pushed to dockerhub.

File: docker-compose/community-docker-compose.yml [hunk 8]
```diff
@@ -10,7 +10,7 @@ version: "2"
 
 services:
     alfresco:
-        image: alfresco/alfresco-content-repository-community:7.0.0-A20
+        image: alfresco/alfresco-content-repository-community:7.0.0-A27c
```
Comment (alandavis) [inline docker-compose/community-docker-compose.yml:13]: The acs-community-packaging build has not been pushing images to DockerHub for a month. It took three goes to work out why, hence the 'c' suffix to the tag. Enterprise's tag is simply 7.0.0-A27.

**Comment by TheOutsSeeker** [comment]
Thanks @alandavis for reverting the changes made 6.0.N + 6.1.N and pulling the right ACS images for 7.0.N. PR looks good to me.

**Pull request merged**